TombEngine/Shaders/InstancedSpriteBuffer.hlsli

```diff
@@ -0,0 +1,15 @@
+#define MAX_SPRITES 128
+
+struct InstancedSprite
+{
+	float4x4 World;
+	float4 Color;
+	float IsBillboard;
+	float IsSoftParticle;
+	float2 SpritePadding;
+};
+
+cbuffer InstancedSpriteBuffer : register(b13)
```

Comment:
might aswell use a `tbuffer`.
cbuffers are limited to 65535 bytes in size.
`InstancedSprite` is 86 bytes big, which means roughly 780 Sprites can be instanced at once.
For the future it might be worth considering switching to `tbuffer`

Reply:
If I remember correctly, @MontyTRC89 limited every batch of instancing to 100 sprites, so it shouldnt be a problem.